tests/func/metrics/conftest.py

```diff
@@ -0,0 +1 @@
+from tests.func.plots import run_copy_metrics  # noqa: F401
```

Comment:
Created a `run_copy_metrics` fixture, that'd copy files, mostly used as copying temporarily-dumped file into metrics/plots file. Also, as `metrics/params` mostly work with tags and commits, `run_copy_metrics` can set tags and commits itself.